calicoctl/commands/node/run.go

```diff
@@ -141,6 +142,13 @@ Options:
                            within the Docker containers that have been networked
                            by the Calico driver.
                            [default: cali]
+     --use-docker-networking-meta-data
```

Comment:
This is what I have right now
```
Extract the Calico-namespaced Docker container labels
(org.projectcalico.label.*) and apply them to the
container endpoints for use with Calico policy.
When this option is enabled Calico Docker networking
must be enabled, traffic must be explicitly allowed
by configuring Calico policies, and Calico profiles
are not used to enforce network isolation.
```
I'm thinking about dropping the`Calico Docker networking must be enabled' because it is a little obvious and the code has a check that will insure that it is enabled.  How's that sound?